package.json

```diff
@@ -7,24 +7,26 @@
   },
   "dependencies": {
     "babel": "5.1.5",
-    "node-sass": "3.0.0-alpha.0",
-    "react": "0.13.1",
-    "react-router": "0.13.2",
+    "browserify": "9.0.8",
+    "cookies": "^0.5.0",
```

Comment:
When I edit this file manually, I intentionally leave out the ^ so that any upgrades are explicit.